README.md

```diff
@@ -83,6 +85,16 @@ OpenAPM currently supports RED Metrics for NestJS v4 and above.
 openapm.instrument('nestjs');
 ```
 
+### Next.js
+
+OpenAPM supports RED metrics for both pages and app router in a Next.js application.
+
+```js
+openapm.instrument('nextjs');
+```
+
+> Note: You can only use the library if Next.js is running in a Node.js environment. Since OpenAPM relies on prom-client for capturing metrics data a serverless environment might not be able persist them.
+
```

Comment:
```suggestion
> Note: You can only use the library if Next.js runs in a Node.js environment. Since OpenAPM relies on prom-client for capturing metrics data, a serverless environment might be unable to persist them.

```